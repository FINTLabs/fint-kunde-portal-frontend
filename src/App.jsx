import React from "react";
import "./App.css";
import {createTheme, ThemeProvider} from "@mui/material/styles";
import Main from "./main/Main";
import {Provider} from "react-redux";
import store from "./data/redux/store/configure-store";
import {CookiesProvider} from "react-cookie";
import AppProvider from "./data/context/AppProvider";


const theme = createTheme({
    palette: {
        secondary: {
            light: '#7fb434',
            main: '#5FA202',
            dark: '#427101',
        },
        primary: {
            light: '#4b727a',
            main: '#1F4F59',
            dark: '#15373e',
        },

        typography: {
            fontFamily: [
                "Nunito Sans", 'sans-serif'
            ].join(','),
        },
    }
});

function App() {
    return (
        <Provider store={store}>
            <CookiesProvider>
                <AppProvider>
                    <ThemeProvider theme={theme}>
                        <Main/>
                    </ThemeProvider>
                </AppProvider>
            </CookiesProvider>
        </Provider>
    );
}

App.propTypes = {};

export default App;
