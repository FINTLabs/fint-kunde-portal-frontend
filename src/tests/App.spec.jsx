import React from "react";
import ReactDOM from "react-dom";
import App from "../App";
import { create } from "react-test-renderer";

describe("App ", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<App />, div);
  });

  it("should match snapshot", () => {
    const app = create(<App />);
    expect(app.toJSON()).toMatchSnapshot();
  });
});
