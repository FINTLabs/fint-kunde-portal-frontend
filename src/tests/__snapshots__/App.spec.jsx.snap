// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App  should match snapshot 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 NoGoMessage-message-105 MuiPaper-rounded"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
      >
        <img
          alt="logo"
          className="NoGoMessage-logo-106"
          src="/fint.svg"
        />
      </div>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
      >
        Du har ikke opprettet bruker.
      </div>
    </div>
    <a
      aria-disabled={false}
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-fullWidth"
      href="https://registrering.felleskomponent.no"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
    >
      <span
        className="MuiButton-label"
      >
        Trykk her for Ã¥ opprette konto
      </span>
    </a>
  </div>
</div>
`;
